### PR TITLE
Added the updated exclusion reasons

### PR DESCRIPTION
**Pull request by sabrinatoro**

### CODE REVIEW

File: docs/editors-guide/exclusion-reasons.md [hunk 1]
```diff
@@ -3,31 +3,33 @@
 ## Description 
 _This document is a work in progress._ Noted below are reasons for excluding external ontology or terminology terms from Mondo. See enum file [here](https://github.com/monarch-initiative/mondo/blob/master/src/schema/mondo.yaml).
```
Comment (nicolevasilevsky) [inline docs/editors-guide/exclusion-reasons.md:4]:
```suggestion
_This document is a work in progress._ Noted below are reasons for excluding external ontology or terminology terms from Mondo. See enum file [here](https://github.com/monarch-initiative/mondo/blob/master/src/schema/mondo.yaml). The dashes in column 1 are meant to indicate the code is a subtype of the code above.
```

File: docs/editors-guide/exclusion-reasons.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
 # Exclusion Reasons
 
 ## Description 
-_This document is a work in progress._ Noted below are reasons for excluding external ontology or terminology terms from Mondo. See enum file [here](https://github.com/monarch-initiative/mondo/blob/master/src/schema/mondo.yaml).
+_This document is a work in progress._ Noted below are reasons for excluding external ontology or terminology terms from Mondo. See enum file [here](https://github.com/monarch-initiative/mondo/blob/master/src/schema/mondo.yaml). The dashes in column 1 are meant to indicate the code is a subtype of the code above.
```
Comment (nicolevasilevsky) [inline docs/editors-guide/exclusion-reasons.md:4]: @sabrinatoro I added this additional line

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
@sabrinatoro feel free to merge this when it passes.

File: docs/editors-guide/exclusion-reasons.md (hunk 2 above)
Comment (sabrinatoro) [reply docs/editors-guide/exclusion-reasons.md:4]: great point! thank you!

**Pull request merged**